### DISCUSSION TITLE
block for kids

### DISCUSSION THREAD

**Discussion by leresteux** [backface/turtlestitch#135] [Q&A] [answered]
Hello,
i am fabmanager to the fablab'ke in Brussels

for a workshop with children I created simplified blocks whose unit is the "square".
Before programming, the children make a drawing on 5x5mm squared paper.

i work with a local version of turtlestitch and add my blocks at startup.

also this workshop was very cool and i want to "upgrate" it.

questions :
1/ how can I modify the scene grid to obtain a 5*5mm grid?
2/how can I integrate my blocks with the current blocks in my local version and, if possible, at the top of the list?

thanks :)
AN exemple : 
[turtle-stitch-blocKE-v1-4(beta)(1).zip](https://github.com/user-attachments/files/16199234/turtle-stich-blocKE-v1-4.beta.1.zip)

**Comment by backface** [comment]
hi!

> 1/ how can I modify the scene grid to obtain a 5*5mm grid?

the grid is fixed in size at the moment. so unfortunately you can't.

> 2/how can I integrate my blocks with the current blocks in my local version and, if possible, at the top of the list?

In the current snap! and upcoming turtlestitch you can create your own categories, but at the moment you cannot re-order the palette. I guess you are already loading you blocks on startup? If you use "export blocks.." and "import ..." you add the blocks only. you could also modify your local version's startup URL to load it automatically (see: Snap Manual / G. Preloading a Project when Starting Snap! )

**Reply by leresteux** [reply]
Thx u

**Reply by leresteux** [reply]
hi 
i have tried
file:///D:/turtlestitch-2.7.16/turtlestitch-2.7.16/index.html#open:Examples/vee.xml
and it doesn't work...
i have pull my file in the same file of index and tried 
file:///D:/turtlestitch-2.7.16/turtlestitch-2.7.16/index.html#open:my_exemple.xml
no thing ...

**Reply by backface** [reply]
> hi i have tried file:///D:/turtlestitch-2.7.16/turtlestitch-2.7.16/index.html#open:Examples/vee.xml and it doesn't work... i have pull my file in the same file of index and tried file:///D:/turtlestitch-2.7.16/turtlestitch-2.7.16/index.html#open:my_exemple.xml no thing ...

sorry I forgot current browser's CORS  restrictions don't allow for loading local files. If you can, run it behind a simple webserver (if you have python it's as easy as "python -m http.server" in your directory of choice)

https://stackoverflow.com/questions/43493323/cross-origin-request-for-local-file

**Reply by leresteux** [reply, marked as answer]
Ok thank you
